crates/evm/src/tests/queries/estimate_gas_tests.rs

```diff
@@ -364,7 +365,7 @@ fn estimate_gas_with_varied_inputs_test() {
 
     assert_eq!(
         value_transfer_result.unwrap(),
-        Uint::from_str("0x5208").unwrap()
+        Uint::from(MIN_TRANSACTION_GAS + 156)
```

Comment:
I wonder if a simple transfer fee must be MIN_TRANSACTION_GAS=21000 or MIN_TRANSACTION_GAS+l1 fee